Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -849,6 +850,7 @@ void CFrame::DoStop()
         g_pCodeWindow->GetPanel<CBreakPointWindow>()->NotifyUpdate();
       g_symbolDB.Clear();
       Host_NotifyMapLoaded();
+      Core::SetState(state);
```

Comment:
The state is paused before the confirmation and I have to restore it in debug mode when the shutdown has been confirmed. For instance, if Dolphin is paused, it will still be paused. However, if it isn't paused it won't be running again because we paused it previously (before showing the confirmation box).

Basically, the goal was not to unpause it nor pause it but doing the proper one depending on the current state. So it will shutdown correctly if it was running or still be paused if it wasn't. The reason I put it after the debug deinit is to prevent a race-condition that will invalidate iterators. I'll address that deinit issue (that should be done after the shutdown is complete) in another PR.